app/dataSelect.js

```diff
@@ -94,7 +95,9 @@ DataSelect.prototype.addBuildListener = function() {
 				return;
 			}
 			genomeBuildHelper.setCurrentBuild(value);
+			//console.log("build val", value);
```

Comment:
Can remove this debug statement

Reply:
Hi, I have removed all the debug statements which were left in the latest commit.